src/components/home/UserNotifications.jsx

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 Stanford University see LICENSE for license
+
+import React from "react"
+import { useSelector } from "react-redux"
+import {
+  hasUser as hasUserSelector,
+  selectGroups,
+} from "selectors/authenticate"
+
+const UserNotifications = () => {
+  const hasUser = useSelector((state) => hasUserSelector(state))
+  const userGroups = useSelector((state) => selectGroups(state))
+
+  if (!hasUser) return null // nothing to show if not logged in
+  if (userGroups.length) return null // nothing to show if the user is logged in but is in at least one group
```

Comment:
this gives us the easy ability to extend this component later if we wanted to show something here instead, like which groups the user is in, etc.